net/mscp/Portfile

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+
+name                mscp
```

Comment:
After using the `github` portgroup, you can use `github.setup`:

```
github.setup        upa mscp 0.2.1 v
```

Using this will allow you to drop the `name`, `version`, `homepage` and `master_sites` lines, as `github.setup` implicitly sets all of these.